<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppQt1Class</class>
 <widget class="QMainWindow" name="AppQt1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppQt1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Saisir l'IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueEdit_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Saisir le Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMessageButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher le message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status connexion : Déconnecté</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppQt1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AppQt1Class</receiver>
   <slot>onDisplayMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushMessageButton</sender>
   <signal>clicked()</signal>
   <receiver>AppQt1Class</receiver>
   <slot>OnSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDisplayMessageButtonClicked()</slot>
  <slot>DisplayMessageEmpiler()</slot>
  <slot>DisplayMessageDepiler()</slot>
  <slot>DisplayMessageSupprimer()</slot>
  <slot>OnSendMessageButtonClicked()</slot>
 </slots>
</ui>
